feat(переделал модальные окна и добавил модальое окно с ошибкой серве…

Malaglovets opened the pull request:
…ра): добавил адаптив на 360

Malaglovets committed:
feat(переделал модальные окна и добавил модальое окно с ошибкой сервера): добавил адаптив на 360

AnnaKharatova commented on src/CloudSlash.svg:
нужно перенести в папку images

AnnaKharatova committed:
refactor(register): added redirect to server error

AnnaKharatova committed:
fix(modal): refactor modal + fix forms

Переделана модалка ошибки сервера, внесены корректировки. Исправлены ошибки форм в соответствии с
результатами тестирования.

AnnaKharatova committed:
fix(forms): fix errors in password input
